src/GeneralStateTestsFiller/VMTests/vmTests/push0Gas2Filler.yml

```diff
@@ -0,0 +1,107 @@
+# EIP-3855: PUSH0 instruction - test for gas price
+push0Gas2:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentBaseFee: '10'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 10000000
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: 0
+      code: ':yul { sstore(0, call(100000, shr(96, calldataload(0)), 0, 0, 0, 0, 0)) sstore(1, 1) }'
+      nonce: 0
+      storage: {}
+
+    0000000000000000000000000000000000000100:
+      balance: 0
+      # gas push0 sstore gas push1(0x1) sstore
```

Comment:
Potential improvement:
```
# gas push0 gas swap1 swap2 sub sstore 
```
(not sure swaps are correct, but idea is to have `gas` around `push0` only to avoid dependence on `sstore` pricing which is complicated and prone to change)

Reply:
I've updated the tests based on this comment.